scripts/json2synapse.py

```diff
@@ -0,0 +1,217 @@
+import os
+import sys
+import pandas
+import numpy
+import synapseclient
+from synapseclient import Table
+from pandas.io.json import json_normalize
+import logging
+
+syn = synapseclient.login()
+
+# TODO: replace with auto download of raw files in repo
```

Comment:
This requires the users to point to this repo's path on their machine. 
ex. 
ref_string = subprocess.check_output("git ls-remote --tags https://github.com/Sage-Bionetworks/synapseAnnotations",
                                       shell=True)
....
created a dictionary of {'release version' : 'commit ID'} 

Now I'm working on both version 1 and 2 cases  where the hierarchy of files to list is different.